go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/uptrace/bun/driver/pgdriver v1.1.16
 	github.com/waku-org/go-waku v0.8.0
 	github.com/xmtp/go-msgio v0.2.1-0.20220510223757-25a701b79cd3
-	github.com/xmtp/proto/v3 v3.36.1
+	github.com/xmtp/proto/v3 v3.36.3-0.20240111132545-4e1d2b1b2399
```

Comment:
This will be updated to the proper non-sha version after https://github.com/xmtp/proto/pull/126 merges and is released